database/builddatabase.yaml

```diff
@@ -0,0 +1,16 @@
+#Used for extending build with this for services that use a databases. 
+
+#Using this will require deploycomponents.yaml  
+.buildDatabaseWithNpm:
```

Comment:
"buildDatabase" sounds like it will build a database, not that it is a build/test step which has a sidecar database.

"buildDatabaseWithNpm" sounds similar. Do we need the full categorization in both the filename (i.e. the thing that is imported) and in the task name? Could we get away with one.

I do like the namespacing pattern you have with the directory.

Reply:
How does "buildtestingdatabase" sound? 

with buildDatabaseWithNpm --> buildTestingDatabaseWithNpm